llabake/Hello-Books
# Ft search book list 154621989

#### What does this PR do?
This PR implements get books by search using react and redux
#### Description of Task to be completed?
User should be able to search the library by a key word or author or title
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install all the dependency
* Run npm run dev
* visit the port where the webpack-dev-server started
on successful login, enter a search term a result page should be loaded
#### What are the relevant pivotal tracker stories?
154621989
#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/35538209-5375614e-054d-11e8-8110-e6be1c016e58.png)
